src/sauce-testreporter.js

```diff
@@ -281,3 +284,42 @@ exports.sauceReporter = async ({browserName, assets, assetsPath, results, startT
   updateExportedValue(SAUCECTL_OUTPUT_FILE, { jobDetailsUrl, reportingSucceeded: true });
   return true;
 };
+
+const generateJunitFile = (assetsPath, suiteName, browserName) => {
+  let result;
+  try {
+    const xmlData = fs.readFileSync(path.join(assetsPath, `report.xml`), 'utf8');
+    result = convert.xml2js(xmlData, {compact: true, spaces: 4});
+  } catch (err) {
+    console.error(err);
+  }
+  result.testsuite._attributes.id = 0;
+  result.testsuite._attributes.name = suiteName;
+  result.testsuite._attributes.timestamp = (new Date(result.testsuite._attributes.timestamp)).toISOString();
+  result.testsuite.properties = {};
+  result.testsuite.properties.property = [
+    {
+      _attributes: {
+        name: 'platformName',
+        value: process.platform,
+      }
+    },
+    {
+      _attributes: {
+        name: 'browserName',
+        value: browserName,
+      }
+    }
+  ];
+  result.testsuites = {};
+  result.testsuites.testsuite = result.testsuite;
+  delete result.testsuite;
+
+  try {
+    let options = {compact: true, spaces: 4};
+    let xmlResult = convert.js2xml(result, options);
```

Comment:
I guess we can combine options on this line with options on line 292. What do you think?

Reply:
I actually have no idea for the indent here. 2 spaces or 4 spaces are all fine for me. Or I can just use `\t` for the output.

Reply:
My point is that we can set options variable even before line 292 and reuse the same variable on line 320. Value are the same.